packages/renderer/src/lib/onboarding/OnboardingItem.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+import { onDestroy, onMount } from 'svelte';
+import type { OnboardingStepItem } from '../../../../main/src/plugin/api/onboarding';
+import Markdown from '../markdown/Markdown.svelte';
+import type { ContextUI } from '../context/context';
+import { SCOPE_ONBOARDING } from './onboarding-utils';
+export let extension: string;
+export let item: OnboardingStepItem;
+export let getContext: () => ContextUI;
+export let executeCommand: (command: string) => Promise<void>;
+
+const onboardingContextRegex = new RegExp(/\${onboardingContext:(.+?)}/g);
+const globalContextRegex = new RegExp(/\${onContext:(.+?)}/g);
+let html;
+let isMarkdown = false;
+$: buttons = new Map<string, string>();
+const eventListeners = [];
+onMount(() => {
+  const itemHtml = createItem(item);
+  html = itemHtml;
+  const clickListener = e => {
+    if (e.target instanceof HTMLButtonElement) {
+      const buttonId = e.target.id;
+      let command = buttons.get(buttonId);
+      if (command) {
+        executeCommand(command);
```

Comment:
as a promise being executed, we need to add a .catch() at the end of the method to handle error being thrown

This is the rule https://typescript-eslint.io/rules/no-floating-promises/

(due to a bug it was not included/checked in svelte components but it should be fixed soon)

Reply:
catch added